Tanvir04Khan/01-max-udemy/03-login-page/src/App.js

```diff
@@ -1,17 +1,27 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 
-import Header from "./Components/Login/Header";
+import LoginPage from "./Components/Login/LoginPage";
+import HomePage from "./Components/Home/HomePage";
 
 import "./App.css";
 
 function App() {
   const [isLoggedIn, setIsLoggedIn] = useState(false);
 
+  useEffect(() => {
+    if (localStorage.isLoggedIn) {
+      setIsLoggedIn(true);
+    }
+  }, []);
+
+  const authentication = (isUserLoggedIn) => {
+    setIsLoggedIn(isUserLoggedIn);
+  };
+
   return (
     <div className="App">
-      <Header />
-      <h1>Helllo There!</h1>
-      {}
+      {!isLoggedIn && <LoginPage authentication={authentication} />}
+      {isLoggedIn && <HomePage authentication={authentication} />}
```

Comment:
this prop name doesn't serve any purpose. I am not getting what is being done here :(

Reply:
if it deals with authentication, then it should be

`{!isLoggedIn && <LoginPage onAuthenticate={handleAuthentication} />}`

@Tanvir04khan

Reply:
changed code as per suggestion @faran4engg